commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/util/test/CPDefinitionHelperTest.java

```diff
@@ -285,44 +241,12 @@ public void testSearchStar() throws PortalException {
 		return cpInstances;
 	}
 
-	private SearchContext _getSearchContext(String keywords, int status) {
-		SearchContext searchContext = new SearchContext();
-
-		LinkedHashMap<String, Object> params = new LinkedHashMap<>();
-
-		if (Validator.isNotNull(keywords)) {
-			params.put("keywords", keywords);
-		}
-		else {
-			params.put("keywords", StringPool.STAR);
-		}
-
-		Map<String, Serializable> attributes = new HashMap<>();
-
-		attributes.put(Field.STATUS, status);
-		attributes.put("params", params);
-
-		searchContext.setAttributes(attributes);
-
-		searchContext.setCompanyId(_group.getCompanyId());
-		searchContext.setGroupIds(new long[] {_group.getGroupId()});
-
-		if (Validator.isNotNull(keywords)) {
-			searchContext.setKeywords(keywords);
-		}
-		else {
-			searchContext.setKeywords(StringPool.STAR);
-		}
-
-		return searchContext;
-	}
-
 	private static final int _CP_INSTANCES_COUNT = 10;
 
 	@Inject
 	private CPDefinitionHelper _cpDefinitionHelper;
 
-	@DeleteAfterTestRun
+	//@DeleteAfterTestRun
```

Comment:
Should this be commented out? Seems like it should either be removed, or actually used.